daemon/response.go

```diff
@@ -150,7 +151,7 @@ func ErrorResponse(status int) ErrorResponseFunc {
 		Status: status,
 	}
 
-	return r.SetError
+	return r.setError
```

Comment:
how about this

On 21 January 2016 at 14:19, Samuele Pedroni <notifications@github.com>
wrote:

> In daemon/response.go
> <https://github.com/ubuntu-core/snappy/pull/352#discussion_r50405261>:
>
> > @@ -150,7 +151,7 @@ func ErrorResponse(status int) ErrorResponseFunc {
> >  		Status: status,
> >  	}
> >
> > -	return r.SetError
> > +	return r.setError
>
> @chipaca <https://github.com/chipaca> or also MakeErrorResponder ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ubuntu-core/snappy/pull/352/files#r50405261>.
>